src/site/content/en/blog/file-system-access/index.md

```diff
@@ -53,35 +53,16 @@ familiar to you. I encourage you to read it anyway, because not all systems are
 API to ensure that people can easily manage their files. See the
 [security and permissions](#security-considerations) section for more information. {% endAside %}
 
-## Current status {: #status }
-
-<div>
-
-| Step                                     | Status                |
-| ---------------------------------------- | --------------------- |
-| 1. Create explainer                      | [Complete][explainer] |
-| 2. Create initial draft of specification | [Complete][spec]      |
-| 3. Gather feedback & iterate on design   | [Complete][spec]      |
-| 4. Origin trial                          | Complete              |
-| **5. Launch**                            | **Complete**          |
-
-</div>
```

Comment:
I think this table should remain. All our articles have it, even if all lines are marked as **complete**.

Reply:
It’s mostly consistency with all other articles.